applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp

```diff
@@ -130,9 +130,12 @@ namespace Kratos
 		const double A = this->GetProperties()[CROSS_AREA];
 		const double L = this->CalculateReferenceLength();
 
-		const double J = this->GetProperties()[IT];
-		const double Iy = this->GetProperties()[IY];
-		const double Iz = this->GetProperties()[IZ];
+		Matrix inertia = GetProperties()[LOCAL_INERTIA_TENSOR];
```

Comment:
OK, but that variable maybe was designed for rigid elements, I did a grep and it is used in the rigid element from the ContactMechanicsApplication and in the legacy structual. In this one the values are stored in a traditional way:

~~~cpp
    Matrix* inertia;
    if( GetProperties().Has(LOCAL_INERTIA_TENSOR) )
    {
        inertia = &(GetProperties()[LOCAL_INERTIA_TENSOR]);
    }
    else if( GetProperties().Has(INERTIA) )
    {
        inertia = &(GetProperties()[INERTIA]);
    }
    else if( Has(LOCAL_INERTIA_TENSOR) )
    {
        inertia = &(GetValue(LOCAL_INERTIA_TENSOR));
    }
    else if( Has(INERTIA) )
    {
        inertia = &(GetValue(INERTIA));
    }
    else
        KRATOS_THROW_ERROR(std::logic_error, "The Inertia is not fully defined for the element", "")
    mInertia_x = (*inertia)(0,0);
    mInertia_y = (*inertia)(1,1);
    mInertia_Polar = (*inertia)(0,1);
~~~